package.json

```diff
@@ -74,8 +75,15 @@
     "sinon": "^9.0.3",
     "snyk": "^1.381.1"
   },
-  "optionalDependencies": {},
+  "overrides": {
+    "connect-multiparty@^2.2.0": {
```

Comment:
Noted this from the [NPM page](https://www.npmjs.com/package/connect-multiparty):

> I actually recommend against using this module. It's cleaner to use the multiparty API directly.